x-pack/plugins/lens/public/xy_visualization/xy_config_panel/threshold_panel.tsx

```diff
@@ -116,29 +116,35 @@ export const ThresholdPanel = (
   }
 ) => {
   const { state, setState, layerId, accessor } = props;
-  const index = state.layers.findIndex((l) => l.layerId === layerId);
-  const layer = state.layers[index];
+
+  const { inputValue: localState, handleInputChange: setLocalState } = useDebouncedValue<XYState>({
+    value: state,
+    onChange: setState,
```

Comment:
There's a problem here with the `empty string` scenario below: by default `useDebouncedValue` will prevent falsy value to be sent upstream.
Enabling `shouldUpdateWithFalsyValue` here will then propagate the empty string, which is not what we want: the solution is to wrap the `setState` with a validation layer, or add a validation prop to this hook

Reply:
@dej611 fixed, thanks!